apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-content.js

```diff
@@ -32,7 +32,7 @@ function getTourContent( localeSlug ) {
 		{
 			heading: __( 'Everything is a block', 'full-site-editing' ),
 			description: __(
-				'In the WordPress Editor, paragraphs, images, and videos are all blocks.',
+				'In the WordPress Editor, paragraphs, images, videos and audios are all blocks.',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.